tests/frequentist_stats/test_linear_model_functions.py

```diff
@@ -593,6 +597,36 @@ def test_compare_branches_lm(interactive, deallocate):
         assert sorted(out["comparative"][branch].index) == expected_index
 
 
+def test_compare_branches_lm_fallback(caplog):
+    # tests whether the results when called with a missing covariate equal the
+    # results when called without a covariate. Also tests that a log was emitted.
+    msg = (
+        "Covariate fake_covariate not found, falling back to unadjusted inferences",
+    )[0]
```

Comment:
Isn't this just a string?

```suggestion
    msg = "Covariate fake_covariate not found, falling back to unadjusted inferences"
```

Reply:
That's what I thought, and probably what the formatter thought too (wrapping the string in parentheses). But pytest complained about the type of the argument, saying that it got a tuple.

Reply:
I guess not: 
<img width="851" alt="Screenshot 2024-07-08 at 1 25 11 PM" src="https://github.com/mozilla/mozanalysis/assets/2738595/9b873b85-0007-43d0-9dc0-03fc16cd05f2">

Reply:
I think the formatter must have applied the tuple wrapping when the error message was different and longer, because I [was able to](https://github.com/mozilla/mozanalysis/pull/268/commits/70b35c72218781d1f3f29d145646329fe63135bb#diff-ccf4587009a6d1005c829f95f8654a608eaf8124fbc1be967c9d731352887d55R603) just set `msg` to the correct string

Reply:
Ah yea something like that makes sense. Thanks!